datarepo-clienttests/build.gradle

```diff
@@ -8,9 +8,6 @@ plugins {
 
 repositories {
     mavenCentral()
-    maven {
-        url 'https://broadinstitute.jfrog.io/broadinstitute/libs-release-local/'
-    }
```

Comment:
What's the reason for removing this repo?

Reply:
It's no longer used. We used to publish the client here, but now we don't any more.

Reply:
Turns it out was needed after all, for a SAM jar. So much for local testing...